build/documentation/generate-typedocs.js

```diff
@@ -21,20 +21,15 @@ const { exec } = require("child_process");
 const { spawn } = require("child_process");
 
 const rootDir = path.resolve(process.cwd());
-const srcDir = "packages/*";
+const srcDir = "packages/*/*";
 const destDir = path.join("docs", "en", "packages");
 
 var copyReadmeScript = "copy-package-readme.js";
 var copyReadmeScriptPath = path.join("build", "documentation", copyReadmeScript);
 var dryRun = false;
 var verbose = false;
 
-const excludedPackages = [
-    "fast-browser-extensions",
-    "fast-color-explorer",
-    "fast-permutator",
-    "fast-eslint-rules",
-];
+const excludedPackages = ["fast-eslint-rules", "fast-tslint-rules"];
```

Comment:
Why is fast-color-explorer and fast-browser-extensions missing?

Reply:
fast browser extensions does not exist anymore. Fast color explorer is no longer under packages and so it doesn't need to be excluded because it is no longer included.